src/views/dialogs/ReceiveDialog.tsx

```diff
@@ -35,16 +36,16 @@ export const ReceiveDialog = ({ selectedAsset = DydxChainAsset.DYDX, setIsOpen }
       value: DydxChainAsset.USDC,
       label: (
         <Styled.InlineRow>
-          <AssetIcon symbol="USDC" /> USDC
+          <AssetIcon symbol="USDC" /> {usdcLabel}
         </Styled.InlineRow>
       ),
     },
     {
-      value: DydxChainAsset.DYDX,
+      value: DydxChainAsset.CHAINTOKEN,
       label: (
         <Styled.InlineRow>
           {/* <AssetIcon symbol="DYDX" />  */}
```

Comment:
can we add a TODO here to add this back? or just make a conditional like chainTokenImage ? <Image> : undefined

Reply:
Adding a comment for now, will follow up with a conditional